src/main/java/aeneas/controllers/ChangeLevelTypeMove.java

```diff
@@ -1,50 +1,35 @@
 package aeneas.controllers;
 import aeneas.models.Level;
-import aeneas.models.Model;
 
 /**
  * Move action to change the type of a level
- * @author Logan
- *
  */
 public class ChangeLevelTypeMove implements IMove {
 
+  Level newLevel;
 
-  Level level;
-
-  Model model;
-
-  int type;
-
-  /**
-   * Constructor
-   *
-   * @param level The level that is currently being played
-   * @param model The game model that holds the level being changed
-   * @param type the type of the level to be changed to
-   */
-  public ChangeLevelTypeMove(Level level, Model model, int type) {
-    this.level = level;
-    this.model = model;
-    this.type = type;
+  public ChangeLevelTypeMove(Level level) {
+    this.newLevel = level;
   }
 
   @Override
   public boolean execute() {
-    // TODO Auto-generated method stub
-    return false;
+    if (!isValid()) return false;
+
+    return true;
   }
 
   @Override
   public boolean undo() {
```

Comment:
Yea. I don't know what to do with these but they *should* be undoable

On Sun, Apr 24, 2016, 10:06 James Kuszmaul <notifications@github.com> wrote:

> In src/main/java/aeneas/controllers/ChangeLevelTypeMove.java
> <https://github.com/Cheddarpuffs/Aeneas-KabaSuji/pull/54#discussion_r60845159>
> :
>
> >    }
> >
> >    @Override
> >    public boolean execute() {
> > -    // TODO Auto-generated method stub
> > -    return false;
> > +    if (!isValid()) return false;
> > +
> > +    return true;
> >    }
> >
> >    @Override
> >    public boolean undo() {
>
> Do we want this to be non-undoable? Not that you actually use this
> anywhere (in which case we may as well delete it).
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/Cheddarpuffs/Aeneas-KabaSuji/pull/54/files/2ba2558436964a67b761818c1294962cf766161b#r60845159>
>